seed.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Welcome to the seed file! This seed file uses a newer language feature called...
+ *
+ *                  -=-= ASYNC...AWAIT -=-=
+ *
+ * Async-await is a joy to use! Read more about it in the MDN docs:
+ *
+ * https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/async_function
+ *
+ * Now that you've got the main idea, check it out in practice below!
+ */
+const db = require('./server/db')
+const {User} = require('./server/db/models')
+
+async function seed () {
+  await db.sync({force: true})
+  // Whoa! Because we `await` the promise that db.sync returns, the next line will not be
+  // executed until that promise resolves!
+
+  const users = await Promise.all([
+    User.create({email: 'cody@email.com', password: '123'}),
+    User.create({email: 'murphy@email.com', password: '123'})
+  ])
+  // Wowzers! We can even `await` on the right-hand side of the assignment operator
+  // and store the result that the promise resolves to in a variable! This is nice!
+  console.log(`Seeded ${users.length} users`)
+  console.log(`Seeded successfully`)
+  process.exit(0)
+}
+
+// Execute the `seed` function
+// `Async` functions always return a promise, so we can use `catch` to handle any errors
+// that might occur inside of `seed`
+seed().catch(err => {
+  console.error(err.message)
+  console.error(err.stack)
+  process.exit(1)
+})
```

Comment:
Coupled with the comment above, here's how I did this in one of my own seed files:

```js
seed()
.catch(err => {
	console.error(err)
	process.exitCode = 1 // recommended over `process.exit(1)`
})
.then(() => {
	debug('closing db connection')
	db.close()
	debug('db connection closed')
})
```